rsconnect/api.py

```diff
@@ -9,6 +9,19 @@
 from rsconnect.log import logger
 from rsconnect.models import AppModes
 
+_ok_http_status = [
```

Comment:
This seems pretty specific - would it be fair to characterize all 2xx codes as OK?